test/test_transforms_v2_refactored.py

```diff
@@ -2485,3 +2490,230 @@ def test_correctness(self):
                 assert isinstance(out_value, torch.Tensor) and not isinstance(out_value, tv_tensors.TVTensor)
             else:
                 assert isinstance(out_value, type(input_value))
+
+
+class TestCrop:
+    INPUT_SIZE = (21, 11)
+
+    CORRECTNESS_CROP_KWARGS = [
+        # center
+        dict(top=5, left=5, height=10, width=5),
+        # larger than input, i.e. pad
+        dict(top=-5, left=-5, height=30, width=20),
+        # left, right, top, bottom side
```

Comment:
This may be personal but I think it would be more obvious if this comment was:

```suggestion
        # sides: left, right, top, bottom
```